README.md

```diff
@@ -202,6 +202,8 @@ _Supporting components with generic helper code._
 
 * ⚪ [**Public Suffix List**](components/lib/publicsuffixlist/README.md) - A library for reading and using the [public suffix list](https://publicsuffix.org/).
 
+* 🔴 [**Push-Firebase**](components/lib/push-firebase/README.md) - A library for reading and using the [public suffix list](https://publicsuffix.org/).
+
```

Comment:
*A library for reading and using the public suffix list* :)

Reply:
Push notifications are magic, they do more things than you expect! 😅 

Fixing.